cmd/workers/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	NoOfWorkers       = 100
+	NoOfJobsPerWorker = 100000
+)
+
+func main() {
+	ch := make(chan string)
+	Wg.Add(NoOfWorkers)
+	go CloseWorkersChannel(ch)
+
+	for i := 0; i < NoOfWorkers; i++ {
+		go worker(i, NoOfJobsPerWorker, ch)
```

Comment:
```suggestion
	go CloseWorkersChannel(ch)

	for i := 0; i < NoOfWorkers; i++ {
	        Wg.Add(1)
		go worker(i, NoOfJobsPerWorker, ch)
```
[icza](https://stackoverflow.com/questions/65213707/where-to-put-wg-add) says to add in the loop right before the goroutine. 🤷🏻 🤔

Reply:
just saw that thats great, thanks